x/monitoringc/keeper/msg_server_create_client.go

```diff
@@ -51,8 +51,8 @@ func (k msgServer) CreateClient(goCtx context.Context, msg *types.MsgCreateClien
 
 	// add the client ID as verified client ID
 	k.SetVerifiedClientID(ctx, types.VerifiedClientID{
-		ClientID: clientID,
-		LaunchID: msg.LaunchID,
+		ClientIDs: []string{clientID},
```

Comment:
We must add the `clientID` to the list of existing ones (if exist), we should not remove other already verified client IDs